powerline/segments/common/players.py

```diff
@@ -528,3 +528,61 @@ def get_player_status(self, pl):
 
 {0}
 ''').format(_common_args.format('itunes')))
+
+
+class MocPlayerSegment(PlayerSegment):
+	def get_player_status(self, pl):
+		'''Return Music On Console (mocp) player information.
+
+		mocp -i returns current information i.e.
+		    File: filename.format
+		    Title: full title
+		    Artist: artist name
+		    SongTitle: song title
+		    Album: album name
+		    TotalTime: 00:00
+		    TimeLeft: 00:00
+		    TotalSec: 000
+		    CurrentTime: 00:00
+		    CurrentSec: 000
+		    Bitrate: 000kbps
+		    AvgBitrate: 000kbps
+		    Rate: 00kHz
+
+		For the information we are looking for we don’t really care if
+		we have extra-timing information or bit rate level. The 
+		dictionary comprehension in this method takes anything in 
+		ignore_info and brings the key inside that to the right info of
+		the dictionary.
+		'''
+		now_playing_str = run_cmd(pl, ['mocp', '-i'])
+		if not now_playing_str:
+			return
+		ignore_info = (
+		    'File', 'TimeLeft', 'Title',
+		    'Bitrate', 'AvgBitrate', 'Rate'
+		)
+		now_playing = dict(
+		    [line.split(': ') 
+		    for line in now_playing_str.split('\n')[:-1]
+		    if line.split(': ')[0] not in ignore_info]
+		)
+		state = _convert_state(now_playing.get('State'))
+		return {
+			'state': state,
+			'album': now_playing.get('Album'),
+			'artist': now_playing.get('Artist'),
+			'title': now_playing.get('SongTitle'),
+			'elapsed': _convert_seconds(now_playing.get('CurrentSec', 0)),
+			'total': _convert_seconds(now_playing.get('TotalSec', 0)),
+		}
+
+
+mocp = with_docstring(MocPlayerSegment(),
+('''Return MOC (Music On Console) player information
+
+Requires mocp command be acessible from $PATH.
```

Comment:
If there are known version requirements it is best to put them to documentation.